assets/blocks/jobs/job-placeholder-list.jsx

```diff
@@ -0,0 +1,43 @@
+
+/**
+ * WordPress Dependencies.
+ */
+const { __, sprintf } = wp.i18n,
+	{ Dashicon } = wp.components;
+
+/**
+ * JobPlaceholder component.
+ */
+export default function JobPlaceholderList( { number, className } ) {
+	let placeholders = [];
```

Comment:
`const` is generally preferable over `let` when possible, which in this case it is :)

Reply:
Good catch. The linter will help a lot for these issues 🙂